src/mobile/src/ui/views/wallet/LanguageSelection.js

```diff
@@ -77,10 +74,11 @@ class LanguageSelection extends Component {
         language: PropTypes.string.isRequired,
     };
 
-    constructor() {
-        super();
-
-        this.languageSelected = currentLanguageLabel;
+    constructor(props) {
+        super(props);
+        this.state = {
+            languageSelected: props.language,
```

Comment:
Ideally, we should avoid props to state.

Reply:
We use props in state in many places. Why should it be avoided?